### ISSUE TITLE
clicking in handsontable does not take focus from a dropdown button

### ISSUE THREAD

**Issue by alexisszabo** [handsontable/handsontable#2961]
I upgraded from handsontable v0.10.5 to 0.20.0 today.

With v0.20.0, when you click on a dropdown button, then click on a cell in the handsontable, the focus stays on the dropdown button. The focus should go to the cell that was clicked on.

Please see example at:

https://jsfiddle.net/aszabo/jd0qLsjn/

To reproduce in the fiddle:
1) Make a selection in the dropdown button
2) Click one or more times in the in the handsontable
3) Try pressing arrow keys to move the selected cell. It will not work because the focus is still on the dropdown button.

Thanks for taking a look!

**Comment by alexisszabo** [comment]
To reproduce in the fiddle:
1) Make a selection in the dropdown button
2) Click one or more times in the in the handsontable
3) Try pressing arrow keys to move the selected cell. It will not work because the focus is still on the dropdown button.

**Comment by mznj21** [comment]
I am also experiencing this issue with v0.20.0; please help

**Comment by mznj21** [comment]
i made a little quick fix.  Not sure if this is the right spot for this but it works (starts at line 18183)...
```javascript
this.eventManager.addEventListener(instance.rootElement, 'mousedown', function(event) {
      this.selectionMouseDown = true;
      this.getActiveEditor().beginEditing(); //Added this so clicking sets focus for copy paste and editing 
    if (!that.isTextSelectionAllowed(event.target)) {
      clearTextSelection();
      event.preventDefault();
      window.focus();
    }
  });
,,,

**Comment by mznj21** [comment]
temporary work around...

```javascript
handsontable.addHook('afterSelection', function (row, col) {
            document.activeElement.blur();
        });
```

**Comment by krzysztofspilka** [comment]
Released in 0.20.1

**Comment by alexisszabo** [comment]
Thank you! It works wonderfully!
